h2/events.py

```diff
@@ -136,6 +136,56 @@ def __repr__(self):
         )
 
 
+class HeadersSent(object):
```

Comment:
Part of me wonders if we should prefix these class names with underscores, just to make *really sure* we don't get confused.